lib/view_component/previewable.rb

```diff
@@ -20,6 +20,14 @@ module Previewable
       # Defaults to +true+ for development environment
       #
       mattr_accessor :show_previews, instance_writer: false
+
+      # Set the entry route for component previews through app configuration:
+      #
+      #     config.view_component.preview_route = "/previews"
+      #
+      # Defaults to +/rails/view_components+ when `show_previews' is enabled
+      #
+      mattr_accessor :preview_route, instance_writer: false
```

Comment:
```suggestion
      mattr_accessor :preview_route, instance_writer: false, default: "/rails/view_components"
```

(see https://apidock.com/rails/Module/mattr_accessor)

Reply:
Nice